zebrad/src/components/sync/end_of_support.rs

```diff
@@ -13,7 +13,7 @@ use zebra_chain::{
 use crate::application::release_version;
 
 /// The estimated height that this release will be published.
-pub const ESTIMATED_RELEASE_HEIGHT: u32 = 2_212_380;
+pub const ESTIMATED_RELEASE_HEIGHT: u32 = 2_240_000;
```

Comment:
Mainnet is currently at 2241K, let's check if we need to update this estimate just before we release:
https://blockchair.com/zcash

Reply:
Done in [bde465b](https://github.com/ZcashFoundation/zebra/pull/7610/commits/bde465bd1dc18270eb7b7f01c04cf0e21ddba87a).